src/app/features/auth/pages/login/login-page.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="flex justify-center items-center h-screen">
```

Comment:
You don't need a root wrapper in your templates. `@Component` decorator has `host` property, which has `class` property, where you can specify a string of classes, that will be added to your component tag.